src/Logary/DataModel.fs

```diff
@@ -183,6 +183,29 @@ type Value =
 module Value =
   open Logary.Internals
   open System.Collections.Generic
+  
+  let rec exceptionToStringValueMap (valueOfObject : obj->Value) (e : exn) =
+    let fields =
+      [ yield "type", String (e.GetType ()).FullName
+        yield "message", String e.Message
+        if e.TargetSite <> null then
+          yield "targetSite", String (e.TargetSite.ToString ())
+        if e.StackTrace <> null then
+          yield "stackTrace", String e.StackTrace
+        if e.Source <> null then
+          yield "source", String e.Source
+        if e.HelpLink <> null then
+          yield "helpLink", String e.HelpLink
+        if e.HResult <> 0 then
+          yield "hResult", Int64 (int64 e.HResult)
+        if e.Data <> null && e.Data.Count > 0 then
+          yield "data", valueOfObject e.Data ]
+
+    Map.ofSeq <|
```

Comment:
I only moved this block of code as-is. Also, I'm almost thinking it does bring some benefit here. Thoughts?